src/libraries/System.Diagnostics.DiagnosticSource/tests/System.Diagnostics.DiagnosticSource.Tests.csproj

```diff
@@ -5,9 +5,14 @@
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'browser'">
```

Comment:
Should this be unconditional? Otherwise, we may want to add extra target to `TargetFrameworks`.

Reply:
I think that on other platforms, those are not need because they are on by default.
But I see this as relevant to particular CI leg run, not that we need another flavor of the DLL.

I'm usually confused about those, but I don't what's wrong with that mental model. Hints ? 
Thanks.

Reply:
The libraries build expects each configuration to produce separate binary. There is `-allConfigurations` command line options that builds all configurations in one build script invocation. The general rule is that the conditions in the project file need to match `TargetFrameworks`.

Reply:
I think there is no harm by enabling those flag unconditionally for this test project.